solutions/usaco-1159.mdx

```diff
@@ -2,17 +2,102 @@
 id: usaco-1159
 source: USACO Silver 2021 December
 title: Connecting Two Barns
-author: Nathan Gong
+author: Nathan Gong, Jesse Choe
 ---
 
 [Official Analysis (C++)](http://usaco.org/current/data/sol_prob2_silver_dec21.html)
 
-## Implementation - DFS + Two Pointers
+## Implementation
 
 **Time Complexity:** $\mathcal{O}(T\cdot (N\log N + M))$
 
 <LanguageSection>
 
+<CPPSection>
```

Comment:
I think an explanation of the solution would also be good before the code because the official sol didn't really specify how binary search would work

Reply:
Ok sure...

Reply:
Up to you, comments also good